pkg/config/config.go

```diff
@@ -1024,6 +1024,10 @@ func InitConfig(config Config) {
 	// Serverless Agent
 	config.BindEnvAndSetDefault("serverless.logs_enabled", true)
 	config.BindEnvAndSetDefault("enhanced_metrics", true)
+	config.BindEnvAndSetDefault("trace_enabled", true)
+	config.BindEnvAndSetDefault("capture_lambda_payload", false)
+	config.BindEnv("dotnet_tracer_home")
+	config.BindEnv("service")
```

Comment:
For all of those I'm not sure this is the best solutions. Are those settings used anywhere ?

If yes: would defining them twice be an issue ? What if they're defined twice with different default or type ?
if no: we should not add arbitrary binding in the env but either remove them from the serverless env or not warn about them explicitly in the code from [9646](https://github.com/DataDog/datadog-agent/pull/9646)

Reply:
`trace_enabled`, `dotnet_tracer_home`, and `service` are not used in the agent, but are needed for other purposes in the serverless environment. I've added them to `knownVars` in the `findUnknownEnvVars` function. See the updated files changed